app/src/main/java/org/simple/clinic/patient/recent/RecentPatient.kt

```diff
@@ -55,12 +55,14 @@ data class RecentPatient(
           LEFT JOIN (
             SELECT MAX(createdAt) latestCreatedAt, T.*
               FROM BloodPressureMeasurement T
+              WHERE deletedAt IS NULL
               GROUP BY patientUuid
           ) LAST_BP ON P.uuid=LAST_BP.patientUuid
           LEFT JOIN (
             SELECT MAX(createdAt) latestCreatedAt, T.*
```

Comment:
Can you specify the fields you need instead of using T.* here as well?

Reply:
done